test/endToEnd/subproblemTest.js

```diff
@@ -144,16 +135,23 @@ function changeScale(browser) {
     .click(moveCeil)
     .assert.containsText(floorLabel, '-400')
     .assert.containsText(ceilLabel, '500')
-    .moveToElement(moveLowerValue, 0, 0)
+    .moveToElement(lowerValueLabel, 0, 0)
     .mouseButtonDown(0)
     .moveToElement(moveFloor, 0, 0)
     .mouseButtonUp(0)
     .assert.containsText(lowerValueLabel, '-400')
-    .moveToElement(moveUpperValue, 0, 0)
+    .moveToElement(upperValueLabel, 0, 0)
     .mouseButtonDown(0)
     .moveToElement(moveCeil, 0, 0)
     .mouseButtonUp(0)
     .assert.containsText(upperValueLabel, '500')
+    .assert.containsText(stepSizeSelector, '10')
+    .click(stepSizeSelector)
+    .assert.containsText(stepSizeSelector + '/option[1]', '100')
+    .assert.containsText(stepSizeSelector + '/option[2]', '10')
+    .assert.containsText(stepSizeSelector + '/option[3]', '1')
+    .click(stepSizeSelector + '/option[3]')
+    .assert.containsText(stepSizeSelector, '1')
     .useCss()
```

Comment:
If we change the step size, shouldn't we also check if the values get updated correctly?

Reply:
Can't get it to work in headless mode